php/class-fieldmanager-select.php

```diff
@@ -120,7 +127,7 @@ public function form_element( $value = array() ) {
 
 		$opts = '';
 		if ( $this->is_repeatable() || $this->first_empty ) {
-			$opts .= '<option value="">&nbsp;</option>';
+			$opts .= '<option value="">' . $this->first_empty_label . '</option>';
```

Comment:
this should be escaped:
```suggestion
			$opts .= '<option value="">' . esc_html( $this->first_empty_label ) . '</option>';
```